lib/stacks/next.js

```diff
@@ -46,14 +46,21 @@ const initForTypeScript = async (root) => {
       return JSON.parse(filtered.join(''));
     });
   const compilerOptions = jsConfigJson.compilerOptions;
-  const { module, ...others } = compilerOptions;
+  const { module, ...otherOptions } = compilerOptions;
   const tsConfigBaseJson = {
-    compilerOptions: others
+    compilerOptions: otherOptions
   };
   await fs.writeJson(sysPath.join(root, 'tsconfig.json'), tsConfigBaseJson, { spaces: 2 });
   await fs.remove(sysPath.join(root, 'jsconfig.json'));
   await fs.remove(sysPath.join(root, 'jsconfig.server.json'));
 
+  const packageJson = await fs.readJSON(sysPath.join(root, 'package.json'));
+  const { scripts, ...otherConfigs } = packageJson;
+  scripts.eslint = 'eslint --fix \"./src/**/*.{ts,tsx}\"';
```

Comment:
I think we can change `eslint --fix \"./src/**/*.{ts,tsx}\"` to `next lint`. What do you think?

Reply:
Created PR to fix it https://github.com/pmndrs/react-three-next/pull/48. I'll delete this line.